prober/prober.go

```diff
@@ -288,6 +293,22 @@ func (p *Probe) loop() {
 		return
 	}
 
+	if p.IsContinuous() {
+		// Probe function is going to run continuously.
+		for {
+			p.run()
+			// Wait and then retry if probe fails. We use the inverse of the
+			// configured negative interval as our sleep period.
+			// TODO(percy):implement exponential backoff.
```

Comment:
For future us: other places in code have used [logtail/backoff](https://github.com/tailscale/tailscale/blob/main/logtail/backoff/backoff.go) for backoff, potentially can use it here as well.